### DISCUSSION TITLE
How to change layouts only vertically

### DISCUSSION THREAD

**Discussion by Akaki-1991** [YetiForceCompany/YetiForceCRM#14874] [How to]
How to change layouts only vertically.
version: 6.1.0
![view](https://user-images.githubusercontent.com/75428743/123539627-c4404f00-d74b-11eb-9b2c-48fd9e995dd9.JPG)

**Comment by Akaki-1991** [comment]
:(

**Reply by vovpff** [reply]
You need to rewrite templates.

**Reply by Akaki-1991** [reply]
Thanks for the feedback.

Can you tell me in more detail which templates?

**Reply by vovpff** [reply]
I think you need to modify a lot of templates in folder /layouts/basic/modules/Vtiger/

**Reply by davide-alghi** [reply]
Hi @Akaki29,
try this ... edit ```layouts/basic/modules/Vtiger/EditViewBlocks.tpl``` and go to **line 133**.
there you'll can see this Smarty code
```HTML
<div class="{if $FIELD_MODEL->get('label') eq "FL_REAPEAT"} col-sm-3
{elseif $FIELD_MODEL->get('label') eq "FL_RECURRENCE"} col-sm-9
{elseif $FIELD_MODEL->getUIType() neq "300"}col-sm-6
```
now, move all "col-sm-*" to "col-sm-12", i.e.
```HTML
<div class="{if $FIELD_MODEL->get('label') eq "FL_REAPEAT"} col-sm-12
{elseif $FIELD_MODEL->get('label') eq "FL_RECURRENCE"} col-sm-12
{elseif $FIELD_MODEL->getUIType() neq "300"}col-sm-12
```

save and check layout in edit view

**Reply by Akaki-1991** [reply]
Thank You

**Reply by davide-alghi** [reply]
@Akaki29,
OSSPassword module overrides that (default) Smarty template, so you have to apply the change to the overriding template too (if you need).

edit ```layouts/basic/modules/OSSPasswords/EditViewBlocks.tpl``` and go to line 104 and move
```HTML
<div class="{if $FIELD_MODEL->getUIType() neq "300"} col-sm-6 form-row align-items-center my-1 mx-0 {else} w-100{/if} js-field-block-column{if $FIELD_MODEL->get('hideField')} d-none{/if}" data-field="{$FIELD_MODEL->getFieldName()}" data-js="container">
```
to
```HTML
<div class="{if $FIELD_MODEL->getUIType() neq "300"} col-sm-12 form-row align-items-center my-1 mx-0 {else} w-100{/if} js-field-block-column{if $FIELD_MODEL->get('hideField')} d-none{/if}" data-field="{$FIELD_MODEL->getFieldName()}" data-js="container">
```

NB ```col-sm-6``` to ```col-sm-12```
